File: Beecrowd/2029 Reservatório de Mel/main.py
import math

while(True):
    try:
        PI = 3.14
        v = float(input())
        d = float(input())

        raio = d / 2

        area = PI * math.pow(raio, 2)
        alt = v / area

        print("ALTURA = %.2f" %alt)
        print("AREA = %.2f" %area)
    except EOFError:
        break

File: Beecrowd/2029 Reservatório de Mel/Main.java
import static java.lang.Math.pow;
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static final double PI = 3.14;

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        while (sc.hasNext()) {
            double v = sc.nextDouble();
            double d = sc.nextDouble();

            double raio = d / 2;
            double area = PI * pow(raio, 2);
            double alt = v / area;

            System.out.println(String.format("ALTURA = %.2f", alt));
            System.out.println(String.format("AREA = %.2f", area));
        }

        sc.close();

    }
}
